components/collections/choose-collection-type.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+    <div v-if="show">
+        <tabs style="m-4" :id="'collections_tab'" v-on:tab-changed="updateCollectionsType">
+            <tab
+                :id="'my-collections'"
+                :label="'My Collections'"
+                :selected="true"
+            >
+            </tab>
+            <tab
+                :id="'team-collections'"
+                :label="'Team Collections'"
+                :selected="false"
```

Comment:
```
                :selected="false"
```

`selected` attribute has a default value `false` defined in `tab.vue`. So there's no need to mention it explicitly. You can remove this LOC.